### PR TITLE
Update install-app-guard.md

### PR DESCRIPTION
**Pull request by bzimnisky**
Update to clarify the licensing requirements

### CODE REVIEW

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 1]
```diff
@@ -43,10 +43,13 @@ To learn more about Office update channels, see [Overview of update channels for
...
 ### Licensing requirements
 
-* Microsoft 365 E5 or Microsoft 365 E5 Security
+* Microsoft 365 E5 Security
+* Microsoft 365 A5 for Faculty
+* Microsoft 365 A5 for Students
 
 > [!NOTE]
 > Microsoft 365 Apps for enterprise with shared computer activation or device-based licensing do not have access to Application Guard for Office.
+> Safe Documents licensing plans allow access to Application Gurad for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/safe-docs?msclkid=279c718bd08811ec93a29310a7e2c785&view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
>
> Safe Documents licensing plans allow access to Application Guard for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](/microsoft-365/security/office-365-security/safe-docs).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bzimnisky, kindly commit the suggested changes. Many thanks.

File: microsoft-365/security/office-365-security/install-app-guard.md (hunk 1 above)
Comment (bzimnisky) [reply microsoft-365/security/office-365-security/install-app-guard.md]: Still needs the host part but agree that the query string can be removed

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@bzimnisky, kindly change the link to a relative link as previously requested. The link will not work on the preview page but it will work in the live document once it is merged. For confirmation, you can look at other links in this doc (and other Microsoft docs). Thanks.

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 2]
```diff
@@ -43,10 +43,13 @@ To learn more about Office update channels, see [Overview of update channels for
...
 ### Licensing requirements
 
-* Microsoft 365 E5 or Microsoft 365 E5 Security
+* Microsoft 365 E5 Security
+* Microsoft 365 A5 for Faculty
+* Microsoft 365 A5 for Students
 
 > [!NOTE]
 > Microsoft 365 Apps for enterprise with shared computer activation or device-based licensing do not have access to Application Guard for Office.
+> Safe Documents licensing plans allow access to Application Gurad for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](/microsoft-365/security/office-365-security/safe-docs).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
>
> Safe Documents licensing plans allow access to Application Guard for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](/microsoft-365/security/office-365-security/safe-docs).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@bzimnisky, kindly commit the final suggested change to fix the typo ("Garud" to "Guard"). The additional blank line is also needed to separate the two notes. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by bzimnisky** [comment]
Anything preventing this from getting merged?

**Comment by yogkumgit** [comment]
@schmurky Gentle reminder.

**Comment by bzimnisky** [comment]
> Kindly confirm if this edit is correct. If we don't want users to have to change passwords, don't we have to check the box instead?

Unclear what this comment refers to. This edit is specific to supported licensing plans for Office in Application Guard.

**Comment by yogkumgit** [comment]
@schmurky

**Comment by bzimnisky** [comment]
Ping @schmurky

**Pull request merged**